### DISCUSSION TITLE
How to filter out traces by path?

### DISCUSSION THREAD

**Discussion by Kralizek** [microsoft/aspire#1390] [Q&A] [answered]
My ASP.NET Core app with Razor Pages emits traces for static content. Is it possible to specify paths that should be ignored?

I also get traces for requests originated by the AWS SSM configuration provider.

Ideally, I'd love the traces to be less noisy:
- only user initiated
- avoid static content

Some screenshots:

<img width="546" alt="image" src="https://github.com/dotnet/aspire/assets/2458439/7f2fe80a-6b00-4e3e-a3b0-dcffae4ac6e5">

<img width="853" alt="image" src="https://github.com/dotnet/aspire/assets/2458439/5836d6dd-cc07-431c-ad7f-2a53b02a0800">

<img width="339" alt="image" src="https://github.com/dotnet/aspire/assets/2458439/98585b47-7adc-4f86-ad83-9356964a8b75">

**Comment by DamianEdwards** [comment]
Opened #1416 for this suggestion, thanks.

**Reply by Kralizek** [reply, marked as answer]
Considering #1416 was closed and #5100 was merged, I guess we can close this one.
